CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
   * [Kernel] Add `is_struct/2` guard
   * [Kernel] Support `map.field` syntax in guards
 
+#### ExUnit
```

Comment:
We usually ask to not change the CHANGELOG because it may have conflicts when merging 🙂

Reply:
Got it - removed!